app/domain/conjur/fetch_trusted_proxies.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
```

Comment:
The file name is fetch_trusted_proxies but it also do delete duplication... 
What is the responsibility of this file?

Reply:
@Tovli good point, i am fetching the list from the db which may contains duplication because we do not prevent it in load policy.
So the answer is 2:
1. Fetch the list from DB
2. Prepare the list for validation process - delete duplication

Reply:
Cannot we prevent a duplication on load policy ? This is a better practice of object modelling, and there is a code which handles such things in conjur-parser

Reply:
@oburstein-hub everything is possible :) time is the problem

Reply:
How long will it take and what is your deadline to submit it ?

Reply:
Please take it with @Tovli and @InbalZilberman, i dont think it is in my scope

Reply:
fwiw i'm ok with calling the class `FetchTrustedProxies` while it also removes the duplications before actually returning the proxies.

Reply:
@nessiLahav - regarding what @oburstein-hub wrote - Please open an issue for any work we won't do due to time so we can do it at a later phase. You don't need me or Inbal for this

Regarding the naming - do we usually use verbs to decribe classes? It feels odd to me as I would expect to have an object that deals with request origin and in it a method to fetch the client ip that part of its work is to validate the proxies
WDYT?

Reply:
we use verbs for command classes.